virttest/utils_misc.py

```diff
@@ -2240,18 +2240,25 @@ def get_qemu_binary(params):
     Get the path to the qemu binary currently in use.
     """
     # Update LD_LIBRARY_PATH for built libraries (libspice-server)
+    qemu_binary = None
     qemu_binary_path = get_path(_get_backend_dir(params),
                                 params.get("qemu_binary", "qemu"))
 
     if not os.path.isfile(qemu_binary_path):
         logging.debug('Could not find params qemu in %s, searching the '
                       'host PATH for one to use', qemu_binary_path)
-        try:
-            qemu_binary = utils_path.find_command('qemu-kvm')
-            logging.debug('Found %s', qemu_binary)
-        except utils_path.CmdNotFoundError:
-            qemu_binary = utils_path.find_command('kvm')
-            logging.debug('Found %s', qemu_binary)
+        QEMU_BIN_NAMES = ['kvm', 'qemu-kvm', 'qemu-system-ppc64',
+                          'qemu_system', 'qemu-system-x86']
```

Comment:
Though it already extends a lot, do we consider also include other platform upstream build? Like check 'qemu-system-${arch}'(should be `qemu-system-x86_64` instead of `qemu-system-x86`?)